tests/Jobs/RemoveDataObjectJobTest.php

```diff
@@ -16,7 +17,7 @@
 class RemoveDataObjectJobTest extends SearchServiceTest
 {
 
-    protected static $fixture_file = '../fixtures.yml'; // phpcs:ignore
+    protected static $fixture_file = '../fixtures.yml'; // @phpcs:ignore
```

Comment:
AFAIK you shouldn't need the `@` - ideally just ignore a specific error

Reply:
I was torn in between following commenting out a specific error since it makes the line of code looks rather long and unpleasant to the eyes. I just felt that this error is not really an error for a Silverstripe developer and should be an acceptable convention rather than following 3rd-party conventions that does not care about Silverstripe.